### DISCUSSION TITLE
@export Vector2D with locked ratio (unlockable)

### DISCUSSION THREAD

**Discussion by feois** [godotengine/godot-proposals#9011] [Editor]
![Screenshot_20240202_165947](https://github.com/godotengine/godot-proposals/assets/68548170/2af9e9aa-1098-4f2c-ac89-7f21e599a8f5)

basically this

**Comment by AThousandShips** [comment]
You can do this with [`_get_property_list`](https://docs.godotengine.org/en/stable/classes/class_object.html#class-object-private-method-get-property-list) and using [`PROPERTY_HINT_LINK`](https://docs.godotengine.org/en/stable/classes/class_%40globalscope.html#class-globalscope-constant-property-hint-link), would be interesting with a dedicated export annotation though

Though this will be solved by:
* https://github.com/godotengine/godot/pull/72912

Which allows you to provide a hint
